### PR TITLE
[Task] Implement POST Update Notice Method for Bank #28

### PR DESCRIPTION
**Pull request by CNFloss**

### CODE REVIEW

File: src/bank.ts [hunk 1]
```diff
@@ -88,6 +88,21 @@ export class Bank extends ServerNode {
   // TODO: POST /validator_confirmation_services
 
   // TODO: POST /upgrade_notice
```
Comment (sno2) [inline src/bank.ts:90]: Please remove the upgrade notice todo.

File: src/bank.ts [hunk 2]
```diff
@@ -88,6 +88,21 @@ export class Bank extends ServerNode {
...
+  /**
+   * Sends a signed POST request to the bank for an upgrade notice.
+   * @param nodeIdentifier the node identifier of the bank that is receiving the upgrade notice
+   * @param bankAccount the current bank server's account
+   */
+  async sendUpgradeNotice(nodeIdentifier: string, bankAccount: Account): Promise<any> {
```
Comment (sno2) [inline src/bank.ts:96]: Please remove the return type for the method, sorry that my prompt included it as it is implicit.

File: src/bank.ts [hunk 3]
```diff
@@ -88,6 +88,21 @@ export class Bank extends ServerNode {
...
+    try {
+      await this.postData(
+        "/upgrade_notice",
```
Comment (sno2) [inline src/bank.ts:99]: We don't need to do the `try`-`catch`, please just return the `this.postData` method.

**Review by sno2 — CHANGES_REQUESTED** [review]
Thanks, please review my requests.

File: src/bank.ts [hunk 4]
```diff
@@ -87,21 +87,16 @@ export class Bank extends ServerNode {
...
-    try {
-      await this.postData(
-        "/upgrade_notice",
-        bankAccount.createSignedMessage({ bank_node_identifier: nodeIdentifier })
-      );
-    } catch (err) {
-      console.log(err);
-    }
+  async sendUpgradeNotice(nodeIdentifier: string, bankAccount: Account) {
+    await this.postData(
```
Comment (sno2) [inline src/bank.ts:96]: Please return the request.

**Review by sno2 — CHANGES_REQUESTED** [review]
Thanks, just forgot one thing 😉

File: src/bank.ts (hunk 4 above)
Comment (CNFloss) [reply src/bank.ts:96]: gotcha, my bad took it out for the error handling I'll throw it back in.

**Review by sno2 — APPROVED** [review]
LGTM, thank you very much 👍

**Comment by CNFloss** [comment]
forgot my acc#, 57c10f3554872103c9b91e481347c2522dd5a13757831a51b12180c09e2e50ce

**Comment by sno2** [comment]
> forgot my acc#, 57c10f3554872103c9b91e481347c2522dd5a13757831a51b12180c09e2e50ce

👍 Payment team should handle it within a week.

**Pull request merged**